terraform/aws/helm_historyserver.tf

```diff
@@ -0,0 +1,82 @@
+provider "kubernetes" {
+  host                   = aws_eks_cluster.cluster.endpoint
+  cluster_ca_certificate = base64decode(aws_eks_cluster.cluster.certificate_authority[0].data)
+  token                  = data.aws_eks_cluster_auth.cluster.token
+}
+
+data "aws_eks_cluster_auth" "cluster" {
+  name = "${aws_eks_cluster.cluster.name}"
+  depends_on = [
+    aws_eks_cluster.cluster,
+    helm_release.flink_operator
+  ]
+}
+
+data "kubernetes_config_map" "operator_default" {
+  # our `kind: Deployment` for historyserver
+  # needs to use some of the ConfigMap keys/values
+  # from the default operator config so
+  # we handle that reuse here
+  metadata {
+    name = "flink-operator-config"
+    namespace = "default"
+  }
+  depends_on = [
+    aws_eks_cluster.cluster,
+    helm_release.flink_operator
+  ]
+}
+
+locals {
+  # removing lines that start with '#' b/c TF >> helm doesn't like them
+  filtered_log4j_config = join("\n", [
```

Comment:
I'm actually really confused about what's happening here. Are we copy pasting the default values from a configmap generated by the operator onto our helm setup, but with `#` removed? Why are we copy pasting them?

Reply:
I had removed these changes but there were committed to some other branch it seems. It's reusing the default `flink-operator-config` now